qemu/tests/verify_panic_status_with_pvpanic.py

```diff
@@ -0,0 +1,45 @@
+import logging
+
+import aexpect
+
+
+def run(test, params, env):
+    """
+    Verify the QMP even with -device pvpanic when trigger crash,this case will:
+
+    1) Start VM with pvpanic device.
+    2) Check if pvpanic device is exist in guest.
+    3) Trigger crash in guest.
+    4) Check vm status with QMP.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environmen.
+    """
+
+    qmp_command = params["qmp_command"]
+    stop_kdump_command = params["stop_kdump_command"]
+    trigger_crash = params["trigger_crash"]
+    qmp_check_info = params["qmp_check_info"]
+    check_info = params.get("check_info")
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login()
+    qmp_port = vm.monitor
+
+    if check_info:
+        qtree_info = vm.monitor.info("qtree")
+        if params.get("check_info") not in qtree_info:
```

Comment:
You already have `check_info` defined, not need to get it again.
```suggestion
        if check_info not in vm.monitor.info("qtree"):
```